easybuild/easyblocks/f/fftw.py

```diff
@@ -41,7 +41,12 @@
 # Altivec (POWER), SSE (x86), NEON (ARM), FMA (x86_64)
 # asimd is CPU feature for extended NEON on AARCH64
 FFTW_CPU_FEATURE_FLAGS = FFTW_CPU_FEATURE_FLAGS_SINGLE_DOUBLE + ['altivec', 'asimd', 'neon', 'sse']
-FFTW_PRECISION_FLAGS = ['single', 'double', 'long-double', 'quad-precision']
+
+cpu_arch = get_cpu_architecture()
```

Comment:
@d-rdgz I'm not sure it is a good idea to call `get_cpu_architecture()` when this easyblock is imported...

Also, it seems like overkill, just doing the `continue` on `quad` + `POWER` that you do before is sufficient?